plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -273,7 +272,7 @@ static String getParametersString(Map<String, String> parametersMap) {
       resultBuilder.append(entry.getKey());
       resultBuilder.append('=');
       try {
-        resultBuilder.append(URLEncoder.encode(entry.getValue(), "UTF-8"));
+        resultBuilder.append(URLEncoder.encode(entry.getValue(), StandardCharsets.UTF_8.name()));
```

Comment:
This one should probably stay the string literal. The name may not be what we expect.